src/VSCodeExtension/syntaxes/qsharp.tmLanguage.json.v.template

```diff
@@ -50,7 +50,7 @@
         },
         {
           "name": "keyword.other.qsharp",
-          "match": "\\b(new|not|and|or|w\/|private|internal)\\b"
+          "match": "\\b(internal|new|not|and|or|w\/)\\b"
```

Comment:
I wonder if it might make more sense to have it listed where operation is listed as well (meaning they would get the same coloring - not sure how it usually is).